rmf_fleet_adapter/src/rmf_fleet_adapter/TaskManager.cpp

```diff
@@ -1687,6 +1687,10 @@ void TaskManager::_begin_waiting()
   }
 
   if (!_responsive_wait_enabled)
+    if (_waiting)
+    {
+      _waiting.cancel({"Idle behavior updated"}, _context->now());
+    }
     return;
```

Comment:
It looks like this is missing curly braces that should start after `if (!_responsive_wait_enabled)` and end after `return;`.

Reply:
Added in [1fd06be](https://github.com/open-rmf/rmf_ros2/pull/379/commits/1fd06be942d126700d0d03335a9c738059d9db04), thanks for catching it!